### ISSUE TITLE
QuestPDF.Previewer 2024.3.0-rc and .NET8 - can't install package

### ISSUE THREAD

**Issue by ZANZAAA** [QuestPDF/QuestPDF#833]
Impossible to install the package `QuestPDF.Previewer 2024.3.0-rc` in any project targeting donet8.0

![image](https://github.com/QuestPDF/QuestPDF/assets/45053603/6804a1d6-b454-4beb-a004-4f81237ba2ca)

**To Reproduce**
Tested on VS2022 Version 17.8.6
ConsoleApp, Razor App, Blazor Web App

donet8 seems to be supported based on https://www.nuget.org/packages/QuestPDF.Previewer/2024.3.0-rc#supportedframeworks-body-tab

**Comment by scottjohnstone** [comment]
Instead of using nuget package manager like you do with QuestPDF, you must use 
```
dotnet tool install QuestPDF.Previewer --global 
```
for QuestPDF.Previewer.

As further information, when I attempt to use Nuget package manager to install, I get the same result on 2024.3.0 using OOB ASP.NET Core MVC project (no config):
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1202	Package QuestPDF.Previewer 2024.3.0 is not compatible with net8.0 (.NETCoreApp,Version=v8.0). Package QuestPDF.Previewer 2024.3.0 supports: net8.0 (.NETCoreApp,Version=v8.0) / any	WebApplication1	C:\source\repos\WebApplication1\WebApplication1\WebApplication1.csproj	1	

```
.NET 6 and 8 is installed:
![image](https://github.com/QuestPDF/QuestPDF/assets/66835704/c1475f0f-4973-48df-8311-7e791ee68538)
![image](https://github.com/QuestPDF/QuestPDF/assets/66835704/e7712129-9dc5-4979-bbd9-65ffe33a97b5)

Attempt to install via nuget package manager fail if the project is set to .NET 6, .NET 8, or multitarget .NET 6 and .NET 8.

**Comment by MarcinZiabek** [comment]
Indeed, the QuestPDF.Previewer application should be installed as a global dotnet tool, rather than as a dependency in your project. 

@scottjohnstone You are right 😄 This is a correct way of installation:

```shell
dotnet tool install QuestPDF.Previewer --global 
```

**Comment by MarcinZiabek** [comment]
The recommended approach is to use the QuestPDF Companion App, which provides a dedicated installer and does not rely on dotnet Tool API. You can find more details here: [Companion App Usage](https://www.questpdf.com/companion/usage.html). Hope this helps!

As there has been no further activity, I’m closing this issue. If you have a similar request in the future, please feel free to open a new GitHub issue. I truly appreciate your feedback! ❤️
